### ISSUE TITLE
does not work

### ISSUE THREAD

**Issue by sandmanpl** [Samlis/AliasIsolationForEpicGamesStore#2]
After a bunch of 
"
aliasIsolationInjector.exe
AI.exe process not found

The game launches for me but I see no visual effect
and if I try to run it again
"AI.exe process id is 19356
DLL already injected!"

**Comment by Egor179** [comment]
Check again:
1. In the first line, you must specify the path to the game file to be launched - AI.exe, for example:
```
cmd /C "D:\Games\Epic Games\Games\AlienIsolation\AI.exe"
for /l %%x in (1, 1, 150) do aliasIsolationInjector.exe
```
2. Epic Store should already be running, but the game is not running

Watch the video:
https://www.youtube.com/watch?v=ihGguCbaQmc

**Comment by Samlis** [comment]
Hi

Did you download the latest release? 150 injection attempts should be enough for:
1. Epic store to launch (if not already running)
2. The game to launch and the mod to be injected
As Egor179 indicated, in the command line you should see a message about the DLL being injected and on the load screen you should see the Alias Isolation video.

Remember to run the cmd like this:

epicisolation.cmd [path to AI.exe]

or you can hard-code the path INTO the cmd directly on the first line like Egor179 demonstrated above (but you should not need to do this).

Hope you are successful, please let me know if you get it running.

thank you

**Comment by sandmanpl** [comment]
Hi
Thanks for your reply.
I am running from a data SSD but 150 was not enough for my system I had to increase it to 500.
Now it runs great.
I have also installed the lens flare remove mod and now the game looks fantastic.
Thank you for creating this EPIC-store version of the injector.

**Comment by Samlis** [comment]
Fantastic, I will add a note for people to try a higher count or use the option described by the other user where you can run without Epic Store running.
Glad it is working for you now!

**Comment by ancient-animal** [comment]
I wrote a message about playing this game without the Epic Games Store by using the "-EpicPortal" command-line parameter, but we can't load our progress while using that, so ignore it.

This mod works well for me while NOT using the "-EpicPortal" parameter. It didn't work with 150 tries, even though the Epic Games Store was already open and I use ONLY an SSD for everything, but it worked when I set it to 500 tries.

**Comment by ancient-animal** [comment]
I played through the full game with this mod and it worked very well.

I'll share this information:
While the Epic Games Store is syncing the save files (for the "Cloud Save" feature), this mod won't be able to attach itself to the game, so I checked the "Library" tab of the store, and and looked at the "Alien: Isolation" line, and waited a few seconds until the "Syncing" line switched back to "Launch". Then, I started the game with the "epicisolation.cmd" file of this mod, and the mod attached itself correctly.

That problem happened during these situations:

1. When I needed to close the game and quickly re-open it again.
2. Sometimes the store decided to sync the saves again after many hours without using the game, so I needed to close the game, wait for the "Syncing" line to switch to "Launch", and then use the "epicisolation.cmd" file a second time. Sometimes, it started syncing the save data again after I started the game the second time, so I needed to wait for the "Syncing" line to switch to "Launch" again, and then use the "epicisolation.cmd" file a third time.
